schema.yaml

```diff
@@ -37,6 +37,9 @@ mapping:
     type: seq
     required: True
     sequence:
+      - type: any
```

Comment:
Why should we do this check here versus at the actual resource level?

Reply:
We need the bucket arn + the region as listed. One idea is to get rid of the region in the yaml and just use this to populate it at build time...

Reply:
Got it. Populating at build time makes sense, but I think we wouldn't want to do it now since other things might be dependent on the data in the repo directly (which would be missing the region if we went this way).